foundryup/foundryup

```diff
@@ -242,6 +242,8 @@ OPTIONS:
     -C, --commit    Install a specific commit
     -r, --repo      Install from a remote GitHub repo (uses default branch if no other options are set)
     -p, --path      Install a local repository
+    --arch          Install a specific architecture
+    --platform      Install a specific platform
```

Comment:
For both, it'd be cool to give examples of the available arch/platform to install

Reply:
Would `Install a specific architecture (supports amd64 and arm64)` and `Install a specific platform (supports win32, linux, and darwin)` sufficient?

Reply:
Yep!

Reply:
amended!